### ISSUE TITLE
Default ROS version of master branch

### ISSUE THREAD

**Issue by mch5048** [ROBOTIS-GIT/open_manipulator_simulations#7]
The ROS version of master branch seems compatible with ROS-melodic. I think you should specifiy the compatible version on the e-manual.
Those who use ROS-kinetic would face small trouble using gazebo package if they follow the install instruction of e-manual

**Comment by HPaper** [comment]
Hi @mch5048 
Sorry for the late reply, and thanks for using OpenManipulator.
The master branch version of open_manipulator_simulations is successfully worked on ROS-kinetic environment too. What is the problem you are talking about? please tell me.

**Comment by mch5048** [comment]
@gpwhd07 Sorry for the late reply too :+1:  .
Actually, the problem was related to missing PID gain configuration for the gazebo controller.
I had installed all the packages from the master branch, and configured the system for the gazebo simulation.
However, at the time I launched gazebo controller threw errors of missing pid gains for the simulated controller, which should have been specified at the yaml file.
It seems related to this commit :
https://github.com/ROBOTIS-GIT/open_manipulator_simulations/commit/6acba54eb738533a9f19fecd35e4d6cf15a4e65d#diff-6a539fac0eefe3b88e8e6ba5c1fec0e2,

Because the original config file I had at that time had no pid gains, so I had to copied the values from the kinetic-devel branch.

Anyhow, It seems that the problem has been solved.
